lib/widgets/appbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+PreferredSizeWidget backAppBar(context, {required String label}) {
+  return AppBar(
+      title: Text(
+        "$label",
+        overflow: TextOverflow.ellipsis,
+        style: Theme.of(context).textTheme.headline1,
+      ),
+      centerTitle: true);
+}
+
+PreferredSizeWidget actionBackAppBar(context,
```

Comment:
**question**: Is this used anywhere in the app?

Reply:
Sorry, it not used anywhere!! forgot to take it out.